File: JAVA_COURSE/KOTLIN/RobotManager11/src/robotmanager11/RobotManager11.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package robotmanager11;

/**
 *
 * @author LV
 */
public class RobotManager11
{
    public static void main(String[] args) {
        Robot robot1 = new Robot("Test Robot 1");
        Robot robot2 = new Robot("Test Robot 2");
        System.out.println(robot1.toString());
        System.out.println(robot2.toString());
    }
}

File: JAVA_COURSE/KOTLIN/RobotManager11/src/robotkotlin/RobotManager11.kt
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package robotkotlin

/**
 *
 * @author LV
 */
object RobotManager11 {
    @JvmStatic
    fun main(args: Array<String>) {
        val robot1 = Robot("Test Robot 1")
        val robot2 = Robot("Test Robot 2")
        println(robot1.toString())
        println(robot2.toString())
    }
}